src/app.ts

```diff
@@ -0,0 +1,61 @@
+import { MongoDriverFactory } from "./MongoConnectorFactory";
+import { MongoDriver } from "./MongoConnector";
+import * as body from "body-parser";
+import express from "express";
+import { Console } from "console";
+
+const PORT = 3000;
+const app = express();
+
+app.use(body.json());
+
+let taskDataStore;
+MongoDriverFactory.build()
+  .then(async (datastore) => {
+    taskDataStore = datastore;
+  })
+  .catch(e => {
+    throw e;
+  });
+
+  //Gets All Tasks
+app.get("/", (req,res)=> {
+  taskDataStore.listTasks().then((tasks) => {
+    res.send(tasks);
+    console.log(tasks);
```

Comment:
Can you remove these console logs, please?  They are useful for development but end up being a pain later on (especially when everything is deployed to production) when there are a lot of random console logs throughout the app and it's hard to figure out which one is yours.

Reply:
Sounds good. Just updated